Woof/WoofTests/Model/Preferences/Helpers/UserPreferencesStorage.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
```

Comment:
nit: Should we put the file higher in the dir hierarchy to show every part of tests can use it?

Reply:
deleted this file and transferred the functionality from it to another